### PR TITLE
Merge feature.color-4 into main

### PR DESCRIPTION
**Pull request by nex3**
See https://github.com/sass/sass/issues/2831

[skip dart-sass]
[skip sass-embedded]

### CODE REVIEW

File: proposal/shared-resources.d.ts.md
Comment (jathak) [inline proposal/shared-resources.d.ts.md:1]: Is this proposal supposed to bee added in this PR or in a separate one?

File: EMBEDDED_PROTOCOL_CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+## 3.0.0-dev
```
Comment (jathak) [inline EMBEDDED_PROTOCOL_CHANGELOG.md]: Is this supposed to be a -dev version?

File: proposal/shared-resources.d.ts.md
Comment (nex3) [reply proposal/shared-resources.d.ts.md:1]: This is a merge error—the proposal has been long accepted.

File: EMBEDDED_PROTOCOL_CHANGELOG.md (hunk 1 above)
Comment (nex3) [reply EMBEDDED_PROTOCOL_CHANGELOG.md]: Good call, done.

**Review by jathak — APPROVED** [review]

**Pull request merged**